lib/jellyfish/room_service.ex

```diff
@@ -54,9 +54,10 @@ defmodule Jellyfish.RoomService do
     |> Enum.reject(&(&1 == nil))
   end
 
-  @spec create_room(Room.max_peers()) :: {:ok, Room.t()} | {:error, :bad_arg}
-  def create_room(max_peers) do
-    GenServer.call(__MODULE__, {:create_room, max_peers})
+  @spec create_room(Room.max_peers(), Room.enforce_encoding()) ::
```

Comment:
Is this spec valid? `Room.enforce_encoding` is an atom, and this function takes the encoding as a string.